### ISSUE TITLE
Referencing a token from an inactive set doesn't result in a broken link in the plugin

### ISSUE THREAD

**Issue by rbosker** [tokens-studio/figma-plugin#3186]
**Describe the bug**
Referencing a token from an inactive set doesn't result in a broken link in the plugin

**To Reproduce**
1. Have a light and dark set with a color token.
2. In a new set, reference the created colors from light and dark
3. Make dark or light active and see that the color in the new set is still there (while it should show a broken link).

**Expected behavior**
If a set is not active and reference a token from that set, you expect the link to that token to be broken. 

**Note**
This worked before the 2.1.0 release properly, but probably [this fix](https://github.com/tokens-studio/figma-plugin/commit/0bc599e0c3e6507f721e59400cda42c2a38b7034) introduced the issue.
Related issue: https://github.com/tokens-studio/figma-plugin/issues/3179

**Comment by six7** [comment]
yeah this was changed recently on purpose - inactive sets are now considered even if they are disabled, but are ranked last. this was made so that less users run into that problem of broken references when in fact the set was just not enabled
